cmd/mkpkg/main.go

```diff
@@ -45,6 +45,8 @@ func parseEmptyDirs(s string) []string {
 
 func main() {
 	out := getopt.StringLong("out", 'o', "", "output file to write")
+	name := getopt.StringLong("name", 'n', "tailscale", "package name")
```

Comment:
Please don't use getopt. I thought we'd removed it everywhere. This might be the only last user in any of our trees?

Reply:
Which package should I replace it with? I can make a followup PR to do that.

Reply:
This program's needs are very simple, so just the stdlib `flag` package.

For commands that need complex subcommands and stuff, atm we use ffcli, but I don't think you should use it for this program.

I agree that should be a followup change though.

Reply:
the standard library one we use everywhere else